codex-ui/src/vue/components/checkbox/Checkbox.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div
+    :class="[$style.checkbox, disabled ? $style[`checkbox--disabled`] : '', checkboxValue ? $style[`checkbox--on`] : '']"
+    v-on="disabled === false ? { click: changeValue } : {}"
+  >
+    <svg
+      class="checkmark"
+      xmlns="http://www.w3.org/2000/svg"
+      width="20"
+      height="20"
+      viewBox="0 0 20 20"
+      fill="none"
+    >
+      <path
+        d="M5.99998 9.5L8.8823 12.6705C8.96165 12.7578 9.09892 12.7578 9.17828 12.6705L14.3333 7"
```

Comment:
Seems hardcoded, is there other ways to solve it?

Reply:
can be imported from codex the icons package